eunkyung/app.js

```diff
@@ -0,0 +1,53 @@
+const http = require('http')
+const express = require('express')
+const cors = require('cors')
+const dotenv = require('dotenv')
+const morgan = require('morgan')
+
+dotenv.config()
+
+const { DataSource } = require('typeorm')
```

Comment:
1. 첫번째 http 모듈을 사용하고 있지 않습니다! 삭제해주세요!
2. dotenv 부분은 저번 시간에 설명드린 것과 같이 제일 최상단에서 호출할 수 있도록 수정해주세요!
3. DataSource를 호출하는 부분은 공백을 제거해주세요! 이유는 아래와 같습니다.

모듈을 호출할 때 가독성을 고려하여 작성할 수 있습니다. 순서는 다음과 같습니다:
1. Built-in 모듈 (내장 모듈)
2. Third-party 모듈
3. Custom 모듈
현재 코드 상에서 built-in 모듈에 해당하는 것은 `http` 모듈이 있지만, 현재 코드 상에서 사용하지 않으니 삭제해주세요.
custom 모듈은 앞으로 layered-pattern을 적용하면서 다양한 모듈을 작성할 수 있는데, 그 때 사용할 수 있습니다. 예를 들어 인증된 유저임을 확인할 수 있는 validate 함수를 불러올 때 사용할 수 있습니다.

```suggestion
require("dotenv").config()

const express = require('express')
const cors = require('cors')
const morgan = require('morgan')
const { DataSource } = require('typeorm')
```